blog-posts/the-modern-code-reviewer-2021.md

```diff
@@ -0,0 +1,219 @@
+- You are a seasoned developer and you make sure to fulfill your duties. Sometimes it's also easier **just to do the review** instead of reminding the team again. This brings you into the situation that you are constantly busy with reviews and missing the crucial ones due to a lack of time.
+- Your day is **full of meetings** while your colleagues are programming one new feature after the other. Although you use the breaks between the meetings to check mails and reading the review requests you will never ever be able to check the actual pull request. You trust your peers to review and you get back to your meeting.
+
+### **Lack of knowledge**
+
+You can observe a lack of knowledge and confidence usually among new team members, inexperienced colleagues or in general insecure persons.
+
+- You're not **fully involved** in the topic, therefore you don't feel suited to review the code.
+- You're **new to the team**. and you don't think it's appropriate to provide reviews in a codebase you don't fully understand.
+
+### **Lack of confidence**
+
+- You are an **inexperienced developer** and the things you see in a PR, they just scare you.
+- You feel intimidated. You don't feel comfortable reviewing a given PR, because the PR is created by a seasoned colleague with a much **higher level.** Your feedback is not worth nor expected.
+- It's a super **complex PR**. I don't even understand the problem it's solving. I feel **intimidated** and leave the fun to others.
+- You're **new to the team**. It's **intimidating** to review and potentially criticize code of my new colleagues.
+
+### **Lack of accessibility**
+
+- There is either **no PR description** or just a single sentence for a rather complex change. You are unable to comprehend the changes with the lack of context given or at least some instructions to follow and to check. That's why you **procrastinate** and secretly hope for someone else to fulfil the review duty.
+- An opened PR is **massive in size**, so you **procrastinate** that huge task for days.
+
+### **Lack of empathy**
+
+- You don't **feel other people's pain** when waiting for a PR review. Instead you focus on your work, create your own PR and expecting people to jump in the review right away.
+- There are plenty of PRs open but you have your own important and urgent code changes. Instead of participating in the review to clear the review jam you have to create yet another one**.** In addition you explicitly ask people to review your PR to move to the front of the queue. ****You do this because otherwise your PR will lay around for days. Instead of solving or discussing the problem with the team you are **selfish and unfair** to your colleagues by trying to get your own stuff covered while ignoring the rest.
```

Comment:
not sure what exactly should be emphasized in bold here, but the markdown is probably not as it should be
![image](https://user-images.githubusercontent.com/68643437/134215778-d444dbf3-9016-426d-82d0-1723d1e10906.png)

Reply:
I think you started the review on an outdated commit, maybe because the window was open during the day? Because I pushed an updated a 1-2 hours after I posted the message in Slack. I fixed this stuff in there, thanks for catching it here 🙌